extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```
@@ -1552,4 +1556,123 @@ describes.sandboxed('amp-ad-network-doubleclick-impl', {}, () => {
+        impl.win.ampAdPageCorrelator = 12345;
+      });
+    });
+
+    it('clears if in experiment', () => {
+      forceExperimentBranch(impl.win, CORRELATOR_CLEAR_EXP_NAME,
+          CORRELATOR_CLEAR_EXP_BRANCHES.EXPERIMENT);
+      impl.buildCallback();
+      expect(onVisibilityChangedHandler).to.be.ok;
+      onVisibilityChangedHandler();
+      expect(impl.win.ampAdPageCorrelator).to.not.be.ok;
+      expect(isInExperiment(element, CORRELATOR_CLEAR_EXP_BRANCHES.CONTROL))
+          .to.be.false;
+      expect(isInExperiment(element, CORRELATOR_CLEAR_EXP_BRANCHES.EXPERIMENT))
+          .to.be.true;
+    });
+
+    it('does not clear if in control', () => {
+      forceExperimentBranch(impl.win, CORRELATOR_CLEAR_EXP_NAME,
+          CORRELATOR_CLEAR_EXP_BRANCHES.CONTROL);
+      impl.buildCallback();
+      expect(onVisibilityChangedHandler).to.be.ok;
+      onVisibilityChangedHandler();
+      expect(impl.win.ampAdPageCorrelator).to.equal(12345);
+      expect(isInExperiment(element, CORRELATOR_CLEAR_EXP_BRANCHES.CONTROL))
+          .to.be.true;
+      expect(isInExperiment(element, CORRELATOR_CLEAR_EXP_BRANCHES.EXPERIMENT))
+          .to.be.false;
+    });
+
+    it('does not clear if in neither branch', () => {
+      forceExperimentBranch(impl.win, CORRELATOR_CLEAR_EXP_NAME, null);
+      impl.buildCallback();
+      expect(onVisibilityChangedHandler).to.be.ok;
+      onVisibilityChangedHandler();
+      expect(impl.win.ampAdPageCorrelator).to.equal(12345);
+      expect(isInExperiment(element, CORRELATOR_CLEAR_EXP_BRANCHES.CONTROL))
+          .to.be.false;
+      expect(isInExperiment(element, CORRELATOR_CLEAR_EXP_BRANCHES.EXPERIMENT))
+          .to.be.false;
+    });
+
+    it('attempts to select into branch', () => {
+      impl.buildCallback();
+      expect(onVisibilityChangedHandler).to.be.ok;
+      onVisibilityChangedHandler();
+      expect(
+          impl.win.experimentBranches[CORRELATOR_CLEAR_EXP_NAME] !== undefined);
```

@keithwrightbos 

This test has been silently failing, and I commented it out in https://github.com/ampproject/amphtml/pull/13424

Do we actually expect `impl.win.experimentBranches[CORRELATOR_CLEAR_EXP_NAME]` to be defined here?

(I haven't looked at the code in detail)